grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -526,33 +486,29 @@ class Incidents extends React.Component<IncidentsPageProps, IncidentsPageState>
           }}
           rowKey="pk"
           data={results}
-          columns={columns}
+          columns={this.getTableColumns()}
         />
-        <div className={cx('pagination')}>
-          <CursorPagination
-            current={`${pagination.start}-${pagination.end}`}
-            itemsPerPage={alertGroupStore.incidentsItemsPerPage}
-            itemsPerPageOptions={[
-              { label: '25', value: 25 },
-              { label: '50', value: 50 },
-              { label: '100', value: 100 },
-            ]}
-            prev={prev}
-            next={next}
-            onChange={this.onChangeCursor}
-            onChangeItemsPerPage={this.handleChangeItemsPerPage}
-          />
-        </div>
+        {this.shouldShowPagination() && (
+          <div className={cx('pagination')}>
+            <CursorPagination
+              current={`${pagination.start}-${pagination.end}`}
+              itemsPerPage={alertGroupStore.incidentsItemsPerPage}
```

Comment:
hmm, why don't we use `page_size` from `alertGroupStore.getAlertSearchResult('default');` like


` const { results, prev, next, page_size } = alertGroupStore.getAlertSearchResult('default');`

Reply:
It's exactly the same value